hack/grab-profiles.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SERVER_ADDR=""
+REQUESTED_PROFILES=""
+MEM_PPROF_FLAGS=""
+PROFILE_PATHS=""
+PROFILE_COMPONENTS=""
+OUTPUT_DIR="."
+HEAPSTER_VERSION="v0.18.2"
+TUNNEL_PORT="${TUNNEL_PORT:-1234}"
+
+ARGS=$(getopt -o s:mho: -l server:,output:,inuse-space,inuse-objects,alloc-space,alloc-objects,cpu,master,heapster -- "$@")
+MASTER_PPROF_PATH="debug/pprof"
+HEAPSTER_PPROF_PATH="api/v1/proxy/namespaces/kube-system/services/monitoring-heapster/debug/pprof"
+
+if [ $? -ne 0 ]; then
+  echo "Wrong argument"
```

Comment:
It'a actually a paranoid check for getopt - it's responsible for printing errors in normal cases.